packages/esm-providers-app/src/api/api.ts

```diff
@@ -0,0 +1,40 @@
+import useSWR from 'swr';
+import { Buffer } from 'buffer';
+import { openmrsFetch, restBaseUrl, useConfig } from '@openmrs/esm-framework';
+
+export const BASE_URL = '/ws/rest/v1/provider?v=full';
```

Comment:
```suggestion
const providerUrl = `${restBaseUrl}/provider?v=full`
```
i don't think we need to export this constant since its a simple one linear to define it e.g